README.md

```diff
@@ -70,9 +90,8 @@ Aws::Rails.add_action_mailer_delivery_method(
 )
 ```
 
-Or, if you are storing your AWS keys using Rails 5.2's [Encrypted
-Credentials](http://guides.rubyonrails.org/security.html#custom-credentials),
-use the following initializer code instead:
+Or, if you are using Rails 5.2's Encrypted Credentials, use the following
```

Comment:
I think this section can be dropped now since encrypted credentials are loaded by default, and the example above this one does a good job of showing how to override.

Reply:
I can do that - perhaps I can condense those down to one as appropriate.